### PR TITLE
Add to FAQ

### PR DESCRIPTION
**Pull request by grischard**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -65,6 +65,33 @@ <h2>Where are we now?</h2>
         <div class="about-faqs row pt-2">
           <h2>FAQs</h2>
           <div class="accordion" id="accordionExample">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
```
Comment (nampas) [inline about.html]: The values of `data-bs-target`, `aria-controls` on this line, as well as `id` on line 74 need to all be the same unique string. That's how the bootstrap library knows what clicks correspond to what expands/collapses. `collapseTwo` I think was already used. You can call it anything, e.g. `guillaume-rulez-1`

File: about.html [hunk 2]
```diff
@@ -65,6 +65,33 @@ <h2>Where are we now?</h2>
...
+                  Will the McGuinness Boulevard redesign redirect traffic onto side streets?
+                </button>
+              </h2>
+              <div id="collapseTwo" class="accordion-collapse collapse" data-bs-parent="#accordionExample">
+                <div class="accordion-body">
+                  <p>The McGuinness Boulevard redesign is carefully planned to improve traffic flow and safety without causing significant disruption to side streets. One of the common concerns when a road is redesigned is that traffic will be diverted to smaller side streets, but this is not necessarily the case.</p>
+                  <p></p>Induced demand occurs when an expansion of road capacity leads to more people choosing to drive, which can often negate any improvements in traffic flow. Conversely, when road capacity is reduced, as in the McGuinness Boulevard redesign, some of the traffic may simply evaporate. This is known as traffic evaporation. People may choose alternate routes, switch to public transportation, or change the times they travel.</p>
```
Comment (nampas) [inline about.html]: Any chance we could link to something for "traffic evaporation"? Maybe [this](https://nacto.org/docs/usdg/disappearing_traffic_cairns.pdf)?

File: about.html [hunk 3]
```diff
@@ -117,6 +144,40 @@ <h2 class="accordion-header">
...
                   <p>Although there are some signs on McGuinness indicating it's an evacuation route, specific evacuation routes are no longer in use by NYC. Should an evacuation be necessary, residents will still be able to use the boulevard on foot, by vehicle, or by bicycle.</p>
+                  <p>In Paris, new cycle paths have <a href="https://www-leparisien-fr.translate.goog/amp/paris-75/a-paris-le-delai-dintervention-des-pompiers-a-raccourci-sur-une-urgence-une-minute-ca-joue-02-03-2023-D3MEX74B4NCKHCYROAQBECNVGE.php?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp">significantly reduced</a> emergency vehicle response time.</p>
+                </div>
+              </div>
+            </div>
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                  Businesses are complaining about the lack of outreach. How has DOT consulted the community on the plans?
```
Comment (nampas) [inline about.html]: How about we be a bit more neutral here haha? Maybes something like "I've heard there's been a lack of outreach. How has DOT consulted the community on the plan?"

**Comment by grischard** [comment]
@nampas I think we're ready to go!

**Review by nampas — APPROVED** [review]
yay thanks!

**Pull request merged**